manifests/s/SanDisk/MemoryZone/Desktop/2.1.0.1/SanDisk.MemoryZone.Desktop.installer.yaml

```
@@ -0,0 +1,27 @@
+# Automatically updated by the winget bot at 2025/May/23
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: SanDisk.MemoryZone.Desktop
+PackageVersion: 2.1.0.1
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{4fb47fef-4052-4468-90bc-75bae2761778}_is1'
+ReleaseDate: 2025-03-14
+AppsAndFeaturesEntries:
+- DisplayVersion: 1.0.0

```

```suggestion
AppsAndFeaturesEntries:
- DisplayVersion: 2.1.0
```

By komac: https://github.com/fjwxzde/winget-tools/actions/runs/15385344251/job/43282816902#step:8:17